pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1065,6 +1070,17 @@ func (ctrl *PersistentVolumeController) recycleVolumeOperation(arg interface{})
 		glog.V(3).Infof("volume %q no longer needs recycling, skipping", volume.Name)
 		return
 	}
+	pods, used, err := ctrl.isVolumeUsed(newVolume)
+	if err != nil {
+		glog.V(3).Infof("error listing pods for volume %q: %v", volume.Name, err)
```

Comment:
this error might not have anything to do with listing pods

Reply:
rephrased (and reused `msg`)